### ISSUE TITLE
benchmark: simple two-layer neural networks with Numpy -Test accuracy:  0.8326

### ISSUE THREAD

**Issue by AcceptedDoge** [zalandoresearch/fashion-mnist#130]
**Fashion-mnist** dataset is a good choice for those who want to implement and apply a Two layer neural network classifier without using deep learning frameworks like Pytorch and Tensorflow. So I committed this notebook on [Kaggle](https://www.kaggle.com/accepteddoge/fashion-mnist-with-numpy-neural-networks). Its principle is similar to Stanford CS231n Assignment 1 and finally got the **Test accuracy:  0.8326**. 

The network has the following architecture:

> input - fully connected layer - ReLU - fully connected layer - softmax

The outputs of the second fully-connected layer are the scores for each class.

The final result may not be as good as the output of the deep learning framework...
But it might be helpful for some deep learning beginners instead of using CIFAR-10 or MNIST dataset.
